unit-tests/shared/src/test/scala/javalib/net/InetAddressTest.scala

```diff
@@ -40,9 +42,11 @@ class InetAddressTest {
     assertFalse(all == null)
     assertTrue(all.length >= 1)
 
-    // TODO remove filter on main
-    for (alias <- all; if alias.isInstanceOf[Inet4Address])
-      assertTrue(alias.getCanonicalHostName().startsWith("localhost"))
+    if (!Platform.isWindows) {
+      // TODO remove filter on main
+      for (alias <- all; if alias.isInstanceOf[Inet4Address])
```

Comment:
Could we refactor this to be a standard `while` loop or use ScalaOps for `exists` or other Scala type operations.

Reply:
Thanks. Is that really necessary for a test?